src/main/frontend/src/components/alerts/Alert.svelte

```diff
@@ -1,13 +1,31 @@
 <script>
 	import { Alert } from 'svelte-materialify/src';
-	export let link = "";
+	export let name;
+	export let link;
+	$: headingText = link ? `Define an ${name}` : `Choose an ${name}`;
```

Comment:
a/an grammar is different, you could keep it that way and add a prop `an` which defaults to false.
an organization
a unit
a context
a schema
a schema version